addons/website/static/src/snippets/s_process_steps/000.scss

```diff
@@ -1,9 +1,51 @@
+@mixin line($top, $right) {
+    width: 50%;
+    height: 50%;
+    @include media-breakpoint-up(lg) {
+        z-index: -1;
+        border-top: 1px solid gray("500");
+        @include o-position-absolute($top, $right);
```

Comment:
There is no official order for all scss properties but in general I'd prefer position absolute as the first one: https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines#generic-rules